index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+  integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+  crossorigin="anonymous" referrerpolicy="no-referrer" />
+ <link rel="stylesheet" href="style.css">
+ <title>My prortfolio</title>
+</head>
+
+<body>
+ <header>
+  <div class="navbar">
+   <h1 class="logo">My logo</h1>
+   <div class="navbar"><i class="fa-solid fa-bars"></i></div>
+  </div>
```

Comment:
- [ ] This should be your page's toolbar, and you should use proper HTML tags to wrap the elements for better semantics and increase the SEO of your page. Kindly please use the logo to wrap in the anchor tag instead of the `h1 tag.` :+1: